rasa_core/policies/embedding_policy.py

```diff
@@ -0,0 +1,848 @@
+                                       (None, dialogue_len,
+                                        None, Y.shape[-1]),
+                                       name='b')
+            self.c_in = tf.placeholder(tf.float32,
+                                       (None, dialogue_len,
+                                        extras.shape[-1]),
+                                       name='c')
+            self.is_training = tf.placeholder_with_default(False, shape=())
+
+            self.sim_op, loss, mask = self._create_tf_graph(self.a_in,
+                                                            self.b_in,
+                                                            self.c_in)
+
+            self.train_op = tf.train.AdamOptimizer().minimize(loss)
+
+            # train tensorflow graph
+            self.session = tf.Session()
+
+            self._train_tf(X, Y, extras, helper_data,
+                           loss, mask)
+
+    def continue_training(self, training_trackers, domain, **kwargs):
+        # type: (List[DialogueStateTracker], Domain, **Any) -> None
+        """Continues training an already trained policy."""
+
+        batch_size = kwargs.get('batch_size', 5)
+        epochs = kwargs.get('epochs', 50)
+
+        num_samples = batch_size - 1
+        num_prev_examples = len(training_trackers) - 1
+        for _ in range(epochs):
+            sampled_idx = np.random.choice(range(num_prev_examples),
+                                           replace=False,
+                                           size=min(num_samples,
+                                                    num_prev_examples))
+            trackers = [training_trackers[i]
+                        for i in sampled_idx] + training_trackers[-1:]
+            training_data = self.featurize_for_training(trackers,
+                                                        domain)
+            (batch_a, batch_pos_b, batch_c, actions_for_b
+             ) = self._prepare_data_for_training(training_data)
+
+            batch_b = self._create_batch_b(batch_pos_b, actions_for_b)
+
+            # fit to one extra example using updated trackers
+            self.session.run(self.train_op,
+                             feed_dict={self.a_in: batch_a,
+                                        self.b_in: batch_b,
+                                        self.c_in: batch_c,
+                                        self.is_training: True})
+
+    def predict_action_probabilities(self, tracker, domain):
+        # type: (DialogueStateTracker, Domain) -> List[float]
+        """Predicts the next action the bot should take
+        after seeing the tracker.
+
+        Returns the list of probabilities for the next actions"""
+        if self.session is None:
+            logger.error("There is no trained tf.session: "
+                         "component is either not trained or "
+                         "didn't receive enough training data")
+            return []
```

Comment:
this should still have num actions shape I guess (or be `None` - but not sure this will be handled properly in ensemble)

Reply:
yes, list of zeros, i think